lib/faker/default/id_number.rb

```diff
@@ -80,6 +135,17 @@ def invalid_south_african_id_number
         [id_number, south_african_id_checksum_digit(id_number)].join
       end
 
+      ##
+      # Produces a random Brazilian Citizen Number (CPF).
+      #
+      # @param formatted [Boolean] Specifies if the number is formatted with dividers.
+      # @return [String]
+      #
+      # @example
+      #   Faker::IDNumber.brazilian_citizen_number #=> "53540542221"
+      #   Faker::IDNumber.brazilian_citizen_number(formatted: true) #=> 000.000.000-00
```

Comment:
Maybe the return value could be the same as the above but formatted, what do you think?
```suggestion
      #   Faker::IDNumber.brazilian_citizen_number(formatted: true) #=> "535.405.422-21"
```

Reply:
Makes sense. Thanks!